composer.json

```diff
@@ -11,24 +11,26 @@
         }
     ],
     "require": {
-        "php": "^5.6|^7.0",
-        "egeloen/json-builder": "^2.0|^3.0",
-        "symfony/dependency-injection": "^2.7|^3.0",
-        "symfony/form": "^2.7|^3.0",
-        "symfony/framework-bundle": "^2.7|^3.0"
+        "php": "^5.6 || ^7.0",
```

Comment:
If you are dropping 5.6, you should remove it here too

Reply:
i thought i have to, but it works with SF 3.0 and 3.3, but not with SF 3.1 and 3.2
do not know why.

Reply:
Probably something you have changed have caused it. In #342 I was able to have all versions work so far, but that is because I am doing incremental changes and not going forward until all tests pass. You can either let me finish or try from the ground up to see where the problem is.